ush/forecast_postdet.sh

```diff
@@ -501,14 +501,8 @@ EOF
   JCAP_STP=${JCAP_STP:-$JCAP_CASE}
   LONB_STP=${LONB_STP:-$LONB_CASE}
   LATB_STP=${LATB_STP:-$LATB_CASE}
-
   cd $DATA
-
   affix="nc"
```

Comment:
And this.

Reply:
Variable affix is used locally. Therefore, no need to modify.

Reply:
The point is I want you to replace `$affix` with `"nc"` everywhere it appears. Similarly for the others.

Reply:
since there is not other `affix`, this variable is no longer needed and should be removed as the reviewer suggests.